AmplifyPlugins/Predictions/Podfile

```diff
@@ -1,20 +1,20 @@
 platform :ios, '13.0'
 use_frameworks!
 
-AWS_SDK_VERSION = "2.12.2"
+AWS_SDK_VERSION = "2.12.5"
 
 pod 'SwiftFormat/CLI'
 pod 'SwiftLint'
 
 target 'AWSPredictionsPlugin' do
   pod 'Amplify', :path => '../../'
   pod 'AWSPluginsCore', :path => '../../'
-  pod "AWSTranslate", "~> #{AWS_SDK_VERSION}"
-  pod "AWSRekognition", "~> #{AWS_SDK_VERSION}"
-  pod "AWSPolly", "~> #{AWS_SDK_VERSION}"
-  pod "AWSComprehend", "~> #{AWS_SDK_VERSION}"
-  pod "AWSTranscribe", "~> #{AWS_SDK_VERSION}"
-  pod "AWSTextract", "~> #{AWS_SDK_VERSION}"
+  pod "AWSTranslate", :path => '~/Projects/Amplify/aws-sdk-ios'
+  pod "AWSRekognition", :path => '~/Projects/Amplify/aws-sdk-ios'
+  pod "AWSPolly", :path => '~/Projects/Amplify/aws-sdk-ios'
+  pod "AWSComprehend", :path => '~/Projects/Amplify/aws-sdk-ios'
+  pod "AWSTranscribeStreaming", :path => '~/Projects/Amplify/aws-sdk-ios'
+  pod "AWSTextract", :path => '~/Projects/Amplify/aws-sdk-ios'
```

Comment:
I will haev to remove this before an official PR only keeping the AWS_SDK_Version change